### PR TITLE
Add fetching of Apple Privacy details

### PR DESCRIPTION
**Pull request by kasnder**
Apple has recently added privacy details to its app store. This comments adds this functionality to the plugin (US app store only).

### CODE REVIEW

File: lib/privacy.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const common = require('./common');
+
+async function privacy (opts) {
```
Comment (facundoolano) [inline lib/privacy.js:5]: you use async here but the rest of the codebase is still using promises, can you stick to promises here? (You can send a separate PR if you want to switch the entire codebase to async/await)

**Comment by facundoolano** [comment]
Can you add some testing for the new method?

**Comment by kasnder** [comment]
I don't have so much experience with tests or using Promises instead of async unfortunately.

**Comment by kasnder** [comment]
I've converted this to Promises now.

**Comment by kasnder** [comment]
And also added a test

**Comment by kasnder** [comment]
Any updates? I've fixed all lint errors and added tests.
